src/data/navigation/navigation.json

```
@@ -316,22 +291,61 @@
       }
     }
   },
-  "component-status": {
-    "title": "Component Status"
-  },
   "resources": {
     "title": "Resources"
   },
+  "contributing": {
+    "title": "Contributing",
+    "sub-nav": {
+      "designers": {
+      "title": "Designers"
+    },
+    "developers": {
+      "title": "Developers"
+    }
+  }
+  },
+  "about": {
```

```suggestion
  "updates": {
```

Did we want this section to be updates in the nav ? Will need to update this and also update the labels for the pages inside the section. (they both say about)

---

Yes, updated.

---

updated